### DISCUSSION TITLE
Is it possible to increase the Storage Volume size after first broker deploy?

### DISCUSSION THREAD

**Discussion by eanbiso** [arkmq-org/arkmq-org-broker-operator#856] [Q&A] [answered]
Hi support,
I wanted to understand if it is possible to change the Artemis Persistence Volume size after the first broker deploy, via the Artemis CRD "deploymentPlan.storage.size" field.
I was able to deploy the broker with a new value (e.g. "1Gi"), different from the default one ("2Gi"), but  after the first deploy, any further attempt to change the "deploymentPlan.storage.size" value (moving it, for example, from "1Gi" to "2Gi"), didn't work. Even if looking into the Artemis CR I see the expected value (for example in this case "2Gi"), then looking into the PVC on k8s I still see their first value (in this case "1Gi")  
Is this expected?
Is there any way to force the PVC to be "reallocated", passing to the new value, without loosing their inner data (in case, for example, we want to increase the volume size)?
Many thanks in advance.

**Comment by gaohoward** [comment, marked as answer]
Normally no you need to delete the pvc first. However according to https://kubernetes.io/docs/concepts/storage/persistent-volumes/ you can resize the PVC for certain type of file system. See section **Expanding Persistent Volumes Claims**
